docroot/modules/custom/dct_configurations/src/Form/WelcomeSectionConfigurationForm.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Drupal\dct_configurations\Form;
+
+
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\State\StateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class WelcomeSectionConfigurationForm extends FormBase{
+
+  /**
+   * @var \Drupal\Core\State\StateInterface
+   *   The state service.
```

Comment:
The comment should be first, followed by a new line and after that @var definition.